resources/eventing/charts/controller/templates/webhook-cert-handler.yaml

```diff
@@ -0,0 +1,94 @@
+{{ if .Values.enableNewCRDVersion }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "controller.fullname" . }}-cert-handler
+  labels:
+    app: {{ include "controller.fullname" . }}-cert-handler
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "controller.fullname" . }}-cert-handler
+rules:
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["create", "delete", "get", "patch", "list", "watch", "update"]
```

Comment:
Does it really need all these roles on "customresourcedefinitions"?

Reply:
Good catch. Create and Delete are not needed as the jobs only get, list, patch, update the CRD.